projects/packages/post-list/src/class-post-list.php

```diff
@@ -0,0 +1,110 @@
+<?php
+/**
+ * The PostList Admin Area.
+ *
+ * @package automattic/jetpack-post-list
+ */
+
+namespace Automattic\Jetpack\PostList;
```

Comment:
Could we update `PostList` to `Post_List` to stay with snake case in PHP?

Reply:
Good call! Done! ✅